doc/license.rst

```diff
@@ -1,28 +1,21 @@
 License
 =======
 
-|PyInstaller| is distributed under the `GPL License`_ but with
-an exception that allows you to use it to build commercial products:
+PyInstaller is distributed under a dual-licensing scheme using both the GPL 2.0 License, with
+an exception that allows you to use it to build commercial products - listed below - and the
+Apache License, version 2.0, which only applies to a certain few files. To see which files the Apache
+license applies to, and to which the GPL applies, please see the `COPYING.txt` file which can be
+found in the root of the PyInstaller source repository.
 
- #. You may use PyInstaller to bundle commercial applications out of your
+A quick summary of the GPL license exceptions:
+
+ - You may use PyInstaller to bundle commercial applications out of your
     source code.
 
- #. The executable bundles generated by PyInstaller from your source code
+ - The executable bundles generated by PyInstaller from your source code
```

Comment:
Maybe it's just the cynic in me, but this can be taken in a very wrong way...

PyInstaller does not impose any additional constraints on the generated bundles from the license perspective, but the license conditions of the used python packages still need to be honored.

Reply:
I mean, that's kind of implied. But you're right, and I've added a clarification.